src/Restaurant.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
 
-const Restaurant = () => {
-  const pizzas = 0;
+import Order from "./Order";
+const Restaurant = props => {
   return (
     <div>
       <h3>Restaurant Orders</h3>
       <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <Order {...props} orderType="pizzas" />
```

Comment:
I don't think it causes a problem, but what props are we receiving in this component?

I don't think the *spread* (`{...props}`) is necessary?